### ISSUE TITLE
Probability score

### ISSUE THREAD

**Issue by gccome** [MeRajat/SolvingAlmostAnythingWithBert#1]
Hello @MeRajat , thanks for the great repo! Just wondering if it is possible to also output probability score for the prediction.

**Comment by MeRajat** [comment]
Hi, @gccome, Thanks.  Forward method returns logits also. ```return logits, y, y_hat``` . 👍
